build.gradle

```diff
@@ -30,6 +30,9 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven {
+            url "../../../m2repository/"
```

Comment:
Why do we need this?

Reply:
This slipped in (still need to build against an androidx prerelease. Will remove.